manual/googlechrome-extensions/googlechrome-github-expandinizr/googlechrome-github-expandinizr.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-github-expandinizr</id>
+    <version>1.0.0</version>
```

Comment:
It is, and for actual applications, and I fully agree that this makes sense.  For something that once it is installed, is maintained via the parent application, I don't think it makes sense to try to keep them up to date.